src/formats/bbcode.js

```diff
@@ -715,6 +718,44 @@
 		},
 		// END_COMMAND
 
+		// START_COMMAND: c (inline code)
+		c: {
+			tags: {
+				c: null
+			},
+			isInline: true,
+			allowedChildren: ['#'],
+			format: '[c]{0}[/c]',
+			html: '<span class="inline-code" data-inline-code="1">{0}</span>'
+		},
+		// END_COMMAND
+
+		// START_COMMAND: c (inline code)
+		span: {
+			tags: {
+				span: {
+					'data-inline-code': null
+				}
+			},
+			isInline: true,
+			allowedChildren: ['#'],
+			format: function (element, content) {
+				if (this.opts.allowInlineCode) {
```

Comment:
Remove this condition.
**because:** If it is meant not to be allowed, then it is the website developer's responsibility to remove the BBCode prior to attaching to the textarea.

Reply:
I'll happily remove the configuration option since I always need that feature enabled.
On the other side, the website developer won't have an option to disable that feature.

Its will be either all the code button functionality or nothing.

Reply:
the website developer can delete the BBCode himself. It is in the API documentation.

Reply:
Remove BBCode?! By editing the JavaScript of the SCEditor?!
I can bet 90% of the SCEditor users are probably those who can only follow the instruction as copy&paste and the advanced of them can change the configuration options.
Removing inline code, which is twisted in the code implementation will not be easy.
The options allow to exclude some buttons, so its either no code button at all or have the button for code block and inline code alltogether, which is fine for me since I always need both of the options, and I believe users who add code snippets to their comments will say the same.

I'll remove the option in my next push.

Reply:
> Remove BBCode?! By editing the JavaScript of the SCEditor?!
No. By calling `sceditor.formats.bbcode.remove('c')`